src/centos/7/rpmpkg/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM mcr.microsoft.com/dotnet-buildtools/prereqs:centos-7
+
+# Install the rpm packaging toolchain we need to build rpms
+RUN yum install -y \
+        gcc \
+        make \
```

Comment:
`make` and `sudo` are already installed in the base image.

Reply:
I'll remove them. Didn't seem to make a difference, but want to follow the best practices.